index.html

```diff
@@ -78,5 +78,4 @@ <h2>results</h2>
     
     <script src="./assets/js/script.js"></script>
 </body>
-<script src="./assets/js/script.js"></script>
```

Comment:
Removed this script tag since it makes our code run twice. I think it is also standard to have the script tag before just before the closing body tag